src/Facebook/Facebook.php

```diff
@@ -483,6 +483,27 @@ public function sendBatchRequest(array $requests, $accessToken = null, $graphVer
     }
 
     /**
+     * Instantiates an empty FacebookBatchRequest entity.
+     *
+     * @param  AccessToken|string|null $accessToken  The top-level access token. Requests with no access token
+     *                                               will fallback to this.
+     * @param  string|null $graphVersion The Graph API version to use.
```

Comment:
wrong description alignment :)
BTW, I'd add an empty line between this `@param` and the below `@return`